new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js

```diff
@@ -32,33 +27,28 @@ const FiatBalance = ({
   const editing = isEditing(NAME)
 
   const schema = Yup.object().shape({
-    fiatBalanceCassette1: Yup.number()
+    fillingPercentageCassette1: Yup.number()
       .transform(transformNumber)
       .integer()
       .min(min)
       .max(max)
       .nullable(),
-    fiatBalanceCassette2: Yup.number()
+    fillingPercentageCassette2: Yup.number()
       .transform(transformNumber)
       .integer()
       .min(min)
       .max(max)
       .nullable()
   })
 
-  const fiatBalanceCassette1Percent =
-    (100 * (data?.fiatBalanceCassette1 ?? 0)) / max
-  const fiatBalanceCassette2Percent =
-    (100 * (data?.fiatBalanceCassette2 ?? 0)) / max
-
   return (
     <Formik
       validateOnBlur={false}
       validateOnChange={false}
       enableReinitialize
       initialValues={{
-        fiatBalanceCassette1: data?.fiatBalanceCassette1 ?? '',
-        fiatBalanceCassette2: data?.fiatBalanceCassette2 ?? ''
+        fillingPercentageCassette1: data?.fillingPercentageCassette1 ?? '',
```

Comment:
This is correct but Ramda is great for this. Also it helps us avoid problems with accessing undefined variable properties when network goes down, I had a bug like that on the dashboard page
So before the return statement you can define a variable like this
```suggestion
        const fillingPercentageCassette1 = R.path(['fillingPercentageCassette1'])(data) ?? ''
```
And now in initialValues it becomes { fillingPercentageCassette1, ... }